Update SBRP Version to 1.0.0-beta.20566.2

dseefeld opened the pull request.

dagood reviewed (approved):
FYI this will conflict with https://github.com/dotnet/source-build/pull/1881 due to adjacent line removal in Versions.props.

dseefeld commented:
System.Drawing.Common,4.7.0 included in version 1.0.0-beta.20563.1 was causing issues with tarball build.  Removed from SBRP and updated version here to remove other prebuilts that were included (Microsoft.NETStandard.Library, 1.6.1 & 2.2.0-prerelease.19564.1)
